src/lib/vault.py

```diff
@@ -16,15 +16,11 @@ def __init__(
     token=None,
     role=None,
     kvv2_mount_point=None,
-    path=None,
-    sanitizer=None
```

Comment:
eso had problem with certain values from configmap, required sanitizing. remove.

Reply:
what does the sanitizer do? could this ever cause issues?

Reply:
yeah, looks like sanitizer is only being [used by migration script](https://github.com/artsy/opstools/blob/daac010c034c2126d2756b08d7adf35572288458/src/migrate_config_secrets/migrate_config_secrets/migrate.py#L17). The sanitizers clean up secret values that caused trouble for [ESO](https://external-secrets.io/latest/). So far we found that double quotes and `*` are problematic. We [switched from ESO to custom script](https://github.com/artsy/hokusai-sandbox/pull/73/files#diff-96b99049e96177bf38230f4eca1eb71a04ea98ac9289b032f6108a1b2696eb93L2), so removing sanitizer support from Vault lib should be okay. We can probably remove sanitizer code as well, later on.